test_conformance/non_uniform_work_group/TestNonUniformWorkGroup.cpp

```diff
@@ -706,12 +706,38 @@ size_t TestNonUniformWorkGroup::adjustGlobalBufferSize(size_t globalBufferSize)
   size_t adjustedGlobalBufferSize = globalBufferSize;
   if (deviceMaxAllocObjSize < globalBufferSize) {
     adjustedGlobalBufferSize = deviceMaxAllocObjSize;
-    log_info("globalBufferSize was adjusted from %lu to %lu\n", globalBufferSize, adjustedGlobalBufferSize);
   }
 
   return adjustedGlobalBufferSize;
 }
 
+void TestNonUniformWorkGroup::adjustGlobalSize(size_t maxNumOfGlobalWorkItems)
+{
+    size_t oldNumOfGlobalWorkItems = _numOfGlobalWorkItems;
+    size_t oldGlobalSize[MAX_DIMS];
+    memcpy(oldGlobalSize, _globalSize, sizeof(_globalSize));
+    // Adjusting _globalSize[1] and _globalSize[2] is enough.
+    // _globalSize[0] is unchanged because it controls non-uniform workgroup.
+    for (int i = (int)_dims - 1; i >= 1; --i)
```

Comment:
in such case when _dims == 1 no chance to execute

Reply:
changed to adjust _globalSize[0] too